src/Microsoft.DotNet.Wpf/src/WpfGfx/core/resources/lineargradient.h

```diff
@@ -80,18 +80,19 @@ class CMilLinearGradientBrushDuce : public CMilGradientBrushDuce
         __in_ecount(1) const BrushContext *pBrushContext
         ) const
     {
-                    // If a mapping mode is relative to the brush sizing bounds *and* those
-                    // bounds have changed, then the realization context has changed            
-        return  (   (m_data.m_MappingMode == MilBrushMappingMode::RelativeToBoundingBox) &&
-
-                    // Return true if the brush sizing bounds have changed
-                    //
-                    // We use exact equality here because fuzzy checks are expensive, coming up 
-                    // with a fuzzy threshold that defines the point at which visible changes
-                    // occur isn't straightforward (i.e., the brush sizing bounds aren't
-                    // in device space), and exact equality handles the case we need to optimize
-                    // for where a brush fills the exact same geometry more than once.                
-                    !(IsExactlyEqualRectD(pBrushContext->rcWorldBrushSizingBounds, m_cachedBrushSizingBounds) ));
+        // If a mapping mode is relative to the brush sizing bounds *and* those
+        // bounds have changed, then the realization context has changed.            
+        // If a mapping mode is absolute and we perform transformation operations
```

Comment:
Comments need to be updated to reflect the current implementation.

Reply:
Reverted this change.